### PR TITLE
javadoc completed

### PR DESCRIPTION
**Pull request by DavidOsicka**

### CODE REVIEW

File: catacombs-chimeras-service/src/main/java/cz/muni/fi/service/RoleService.java [hunk 1]
```diff
@@ -13,13 +13,15 @@
      * find role by id
      * @param id ID of role
      * @return role
+     * @throws cz.muni.fi.exceptions.NotFoundException when troop or role not found
```
Comment (Tantalor93) [inline catacombs-chimeras-service/src/main/java/cz/muni/fi/service/RoleService.java]: only role

File: catacombs-chimeras-service/src/main/java/cz/muni/fi/service/RoleService.java [hunk 2]
```diff
@@ -13,13 +13,15 @@
...
      */
     Role findRoleById(final Long id);
 
     /**
      * find role by name
      * @param name of role
      * @return role
+     * @throws cz.muni.fi.exceptions.NotFoundException when troop or role not found
```
Comment (Tantalor93) [inline catacombs-chimeras-service/src/main/java/cz/muni/fi/service/RoleService.java]: only role

File: catacombs-chimeras-service/src/main/java/cz/muni/fi/service/RoleService.java [hunk 3]
```diff
@@ -45,6 +47,7 @@
     /**
      * remove role specified by id
      * @param id of role to delete
+     * @throws cz.muni.fi.exceptions.NotFoundException when troop or role not found
```
Comment (Tantalor93) [inline catacombs-chimeras-service/src/main/java/cz/muni/fi/service/RoleService.java]: only role

File: catacombs-chimeras-service/src/main/java/cz/muni/fi/service/TroopService.java [hunk 4]
```diff
@@ -14,13 +14,15 @@
      * find troop by ID
      * @param id of troop
      * @return troop
+     * @throws cz.muni.fi.exceptions.NotFoundException when troop or role not found
```
Comment (Tantalor93) [inline catacombs-chimeras-service/src/main/java/cz/muni/fi/service/TroopService.java]: only troop

File: catacombs-chimeras-service/src/main/java/cz/muni/fi/service/TroopService.java [hunk 5]
```diff
@@ -46,6 +48,7 @@
     /**
      * remove troop specified by id
      * @param id of troop to delete
+     * @throws cz.muni.fi.exceptions.NotFoundException when troop or role not found
      */
```
Comment (Tantalor93) [inline catacombs-chimeras-service/src/main/java/cz/muni/fi/service/TroopService.java]: only troop

File: catacombs-chimeras-service/src/main/java/cz/muni/fi/service/TroopService.java [hunk 6]
```diff
@@ -61,6 +64,7 @@
      * get all heroes associated with troop
      * @param id of troopId
      * @return list of troop heroes
+     * @throws cz.muni.fi.exceptions.NotFoundException when troop or role not found
```
Comment (Tantalor93) [inline catacombs-chimeras-service/src/main/java/cz/muni/fi/service/TroopService.java]: troop or hero

File: catacombs-chimeras-service/src/main/java/cz/muni/fi/service/RoleService.java (hunk 1 above)
Comment (DavidOsicka) [inline catacombs-chimeras-service/src/main/java/cz/muni/fi/service/RoleService.java]: :D jasny

**Comment by DavidOsicka** [comment]
vše opraveno

**Pull request merged**